### PR TITLE
Added refresh buttons to own groups

### PR DESCRIPTION
**Pull request by Omelianenko**
before:
picture
now:
now

Fixed for Deployment Slots, WebJobs, and LogFiles.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -516,12 +516,12 @@
                 {
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == subFolder",
-                    "group": "1_folderGeneralCommands@2"
+                    "group": "1_folderGeneralCommands@1"
```
Comment (Omelianenko) [inline package.json]: Not sure about this

Comment (EricJizbaMSFT) [reply package.json]: `subFolder` is any folder below the "Files" node. For example "HttpTriggerJS" in my case below:
screen shot 2018-09-06 at 9 25 45 am

I believe "subFolderRefreshCommand" might be more accurate. But honestly the numbers are the only part you _really_ need. The "_subFolderRefreshCommand" part is just for our own readability and I find it more annoying to maintain than helpful. In the the functions repo, I don't use that part at all: https://github.com/Microsoft/vscode-azurefunctions/blob/master/package.json#L373

In other words, I don't really care what put as long as it works :) Please post one screenshot for each case that you changed

File: package.json [hunk 2]
```diff
@@ -531,7 +531,7 @@
                 {
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == webJobs",
-                    "group": "1_webJobsGeneralCommands@2"
+                    "group": "2_webJobsGeneralCommands@1"
```
Comment (Omelianenko) [inline package.json]: webjobsgeneral

File: package.json [hunk 3]
```diff
@@ -516,12 +516,12 @@
...
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == subFolder",
-                    "group": "1_folderGeneralCommands@2"
+                    "group": "1_subFolderRefreshCommands@1"
                 },
                 {
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == logFolder",
-                    "group": "1_folderGeneralCommands@2"
+                    "group": "2_folderGeneralCommands@1"
```
Comment (Omelianenko) [inline package.json]: logfoldergeneral

File: package.json [hunk 4]
```diff
@@ -516,12 +516,12 @@
                 {
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == subFolder",
-                    "group": "1_folderGeneralCommands@2"
+                    "group": "1_subFolderRefreshCommands@1"
```
Comment (Omelianenko) [inline package.json]: subfolderrefresh

File: package.json [hunk 5]
```diff
@@ -426,7 +426,7 @@
                 {
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == deploymentSlots",
-                    "group": "1_deploymentSlotsGeneralCommands@3"
+                    "group": "2_deploymentSlotsGeneralCommands@1"
```
Comment (Omelianenko) [inline package.json]: deployslotgeneral
